Patch104pZH/GameFilesEdited/Data/INI/CommandSet.ini

```
@@ -119,7 +119,7 @@ CommandSet AmericaVehicleChinookCommandSet
   6 = Command_TransportExit
   7 = Command_TransportExit
   8 = Command_TransportExit
-  9 = Command_ChinookUnload
+  9 = Command_EmptyCrawler ; Patch104p @tweak from Command_ChinookUnload, so that all vehicles use same evacuate command button.
```

You can just commit the suggestion. Hardly any work.

```suggestion
  9 = Command_ChinookUnload
```

---

No. This PR is designed to be Merged with Rebase.

---

Care to explain further? We cannot make commits to a branch?

---

Merge with Rebase means the commits are applied to main branch as is. Adding random commits onto the commit stack would break the clean stack.

---

Well that is not an effective workflow and should be rectified.

---

It has some disadvantages but also advantages, aka I can create one change and see all changes work together and test them properly.

---

Creating one change and seeing all changes work together...? What does this mean?

---

I mean creating one branch with all changes to see how they all work together is more valuable in some cases. For example when making particle changes.

---

I still don't understand. Nothing about what you're saying has any relation to making amendment commits to address feedback.